src/client/user/components/Drawer/ControlPanel/LinkAnalytics/index.tsx

```diff
@@ -265,13 +262,13 @@ function Graphs({ data, shortUrl }: GraphsProps) {
             </Hidden>
           </div>
           <Divider />
-          <TabPanel value={tabValue} index={0} dir={theme.direction}>
+          <TabPanel value={tabValue} index={0}>
             <DeviceStatistics deviceClicks={data.deviceClicks} />
           </TabPanel>
-          <TabPanel value={tabValue} index={1} dir={theme.direction}>
+          <TabPanel value={tabValue} index={1}>
             <DailyStatistics dailyClicks={data.dailyClicks} />
           </TabPanel>
-          <TabPanel value={tabValue} index={2} dir={theme.direction}>
+          <TabPanel value={tabValue} index={2}>
```

Comment:
removing this doesn't affect anything? e.g. mobile view

Reply:
it doesn't seem like it affected anything. Both mobile and desktop view looks fine